sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -8,7 +8,9 @@
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
     <DefineConstants>$(DefineConstants);AZURE_NULLABLE;HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461;netcoreapp2.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461;netcoreapp2.1;net5.0;net6.0</TargetFrameworks>
+    <!-- Remove this line when Azure.Core is released for net6.0  -->
```

Comment:
Let's create an issue to track this.

Reply:
Done: https://github.com/Azure/azure-sdk-for-net/issues/32329